mmseg/models/backbones/strideformer.py

```diff
@@ -0,0 +1,765 @@
+        low_feat2 = self.embedding_list[1](low_feat2)
+
+        high_feat_act = F.interpolate(
+            self.act_list[2](self.act_embedding_list[2](inputs[2])),
+            size=low_feat2.shape[2:],
+            mode="bilinear")
+        high_feat = F.interpolate(
+            self.embedding_list[2](inputs[2]),
+            size=low_feat2.shape[2:],
+            mode="bilinear")
+
+        res = low_feat1_act * low_feat2_act * high_feat_act * (
+                low_feat1 + low_feat2) + high_feat
+
+        return res
+
+
+class InjectionMultiSumallmultiallsumSimpx8(nn.Module):
+    def __init__(self,
+                 in_channels=(64, 128, 256, 384),
+                 activations=None,
+                 out_channels=256):
+        super(InjectionMultiSumallmultiallsumSimpx8, self).__init__()
+        self.embedding_list = nn.ModuleList()
+        self.act_embedding_list = nn.ModuleList()
+        self.act_list = nn.ModuleList()
+        for i in range(len(in_channels)):
+            if i != 1:
+                self.embedding_list.append(
+                    ConvBNAct(
+                        in_channels[i], out_channels, kernel_size=1))
+            if i != 0:
+                self.act_embedding_list.append(
+                    ConvBNAct(
+                        in_channels[i], out_channels, kernel_size=1))
+                self.act_list.append(activations)
+
+    def forward(self, inputs):
+        # x_x8, x_x16, x_x32
+        low_feat1 = self.embedding_list[0](inputs[0])
+
+        low_feat2 = F.interpolate(
+            inputs[1], size=low_feat1.shape[-2:], mode="bilinear")
+        low_feat2_act = self.act_list[0](self.act_embedding_list[0](low_feat2))
+
+        high_feat_act = F.interpolate(
+            self.act_list[1](self.act_embedding_list[1](inputs[2])),
+            size=low_feat2.shape[2:],
+            mode="bilinear")
+        high_feat = F.interpolate(
+            self.embedding_list[1](inputs[2]),
+            size=low_feat2.shape[2:],
+            mode="bilinear")
+
+        res = low_feat2_act * high_feat_act * low_feat1 + high_feat
+
+        return res
+
+
+def _make_divisible(v, divisor=8, min_value=None):
+    if min_value is None:
+        min_value = divisor
+    new_v = max(min_value, int(v + divisor / 2) // divisor * divisor)
+    if new_v < 0.9 * v:
+        new_v += divisor
+    return new_v
+
+
+def _create_act(act):
+    if act == "hardswish":
+        return nn.Hardswish()
+    elif act == "relu":
+        return nn.ReLU()
+    elif act is None:
+        return None
+    else:
+        raise RuntimeError(
+            "The activation function is not supported: {}".format(act))
+
+
+@MODELS.register_module()
+class MobileSeg_Base(StrideFormer):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+
+@MODELS.register_module()
+class MobileSeg_Tiny(StrideFormer):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)

```

Comment:
We might add `StrideFormer` to the `MODEL` register and delete these two classes.